middleware-catalog-module/.idea/modules.xml

```diff
@@ -5,12 +5,16 @@
       <module fileurl="file://$PROJECT_DIR$/campus/CampusCode/CampusCode.iml" filepath="$PROJECT_DIR$/campus/CampusCode/CampusCode.iml" />
```

Comment:
This files inside .idea directory should not be here. They are on .ignore. You need to delete them in your machine then make a git add, git commit, git push.